pkg/webhook/tenant/rolebindings_regex.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020-2021 Clastix Labs
+// SPDX-License-Identifier: Apache-2.0
+
+package tenant
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	capsulev1beta1 "github.com/clastix/capsule/api/v1beta1"
+	capsulewebhook "github.com/clastix/capsule/pkg/webhook"
+	"github.com/clastix/capsule/pkg/webhook/utils"
+)
+
+const (
+	ServiceAccountSubject string = "ServiceAccount"
```

Comment:
Let's use the Kubernetes constants: you could import `k8s.io/api/rbac/v1` in order to use the `ServiceAccountKind`.

https://github.com/kubernetes/api/blob/cca52a076791d31b1796453ece4b2f4df97406a2/rbac/v1/types.go#L35